golang/glide.md

```diff
@@ -52,6 +52,21 @@ You can now run `make install_deps` to fetch the dependencies. They will be writ
 
 To add additional dependencies use `glide get <package>`
 
+If the dependency is private, you will need to manually add it to `glide.yaml`, e.g.:
```

Comment:
This seems like a duplicate of the instructions on L86.  Can we chose one place, and update appropriately?